src/sentry/static/sentry/app/plugins/components/issueActions.jsx

```diff
@@ -0,0 +1,333 @@
+import React from 'react';
+
+import {
+  Form,
+  FormState,
+  Select2Field,
+  Select2FieldAutocomplete,
+  TextareaField,
+  TextField
+} from '../../components/forms';
+import GroupActions from '../../actions/groupActions';
+import LoadingError from '../../components/loadingError';
+import LoadingIndicator from '../../components/loadingIndicator';
+import SettingsBase from '../../components/bases/settingsBase';
+import {t} from '../../locale';
+import {defined} from '../../utils';
+
+class IssueActions extends SettingsBase {
```

Comment:
We should rename SettingsBase to something else if we're going to reuse it outside of settings. I'm not sure of a good name, but could just be PluginComponentBase or something